wordle-web/pages/game.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <v-container fluid fill-height class="pa-0">
+    <v-btn outlined absolute top right color="primary" @click="dialog = true">
+      <v-icon primary> mdi-account </v-icon>
+      {{ playerName }}
+    </v-btn>
+
+    <valid-words :wordle-game="wordleGame" />
+    
+    <v-dialog v-model="dialog" width="450" persistent>
```

Comment:
This dialog should appear when the game is over if we don't have a name for a player (e.g. if the name is guest). This could be done with a watcher, or just a function that gets trigged when the game ends.